spaces/S000129/properties/P000205.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000129
+property: P000205
+value: true
+refs:
+- doi: 10.1007/978-1-4612-6290-9_6
+  name: Counterexamples in Topology
+---
+
+{S129|P36}, see item #2 for space #132 in {{doi:10.1007/978-1-4612-6290-9_6}}.
+
+For every $p\in X$ the open segment (radius) $\{\lambda p: 0< \lambda < 1 \}$ is both a closed and open subset of $X\setminus\{p\}$. Hence $p$ is a cut point.
```

Comment:
```suggestion
The space is {P36}; see item #2 for space #132 in {{doi:10.1007/978-1-4612-6290-9_6}}.

And for every $p\in X$ the open segment (radius) $\{\lambda p: 0< \lambda < 1 \}$ is both a closed and open subset of $X\setminus\{p\}$. Hence $p$ is a cut point.
```
I changed the first paragraph, because `{S129|P36}` was a link showing circular reasoning, as you could check on the resulting web page.  It was showing the space was connected by relying on the fact that it was a cut point space, which was itself relying on the fact that it was connected, which was itself relying on ...